bin/meteor.sh

```diff
@@ -96,14 +120,20 @@ EOF
 create_meteor_startup_file() {
```

Comment:
The condition is a bit weird

Why not

If major == 1 → boot.js program.json
else  main.js

and that's it?

Reply:
like what was done on this commit https://github.com/Scalingo/nodejs-buildpack/pull/42/commits/77f725217e27c97f44538bce5656470955884935 ?

Reply:
OOOH you're right I didn't get that the `main.js` was already mentionned (it was hidden from the diff)